docs/api-docs/analytics/bodl-for-storefronts.mdx

```diff
@@ -56,17 +122,25 @@ The `line_item` object has many common fields for browser events, including the
+| <sup>+</sup>`gift_certificate_id` | string | ID of gift certificate. | [Get a cart](/docs/rest-storefront/carts#get-a-cart) <br /> `lineItems.giftCertificates.id` |
+| <sup>+</sup>`gift_certificate_name` | string | Name of gift certificate that appears in the control panel. | [Get a Cart](/docs/rest-storefront/carts#get-a-cart) <br /> `lineItems.giftCertificates.name` |
+| <sup>+</sup>`gift_certificate_theme` | string | Theme of gift certificate. | [Get a cart](/docs/rest-storefront/carts#get-a-cart) <br /> `lineItems.giftCertificates.theme` |
+| <sup>+</sup>`index` | integer | The position of the item in the cart. | [Get a checkout](/docs/rest-storefront/checkouts#get-a-checkout) <br /> `data.cart.lineItems.physicalItems.id` or `data.cart.lineItems.digitalItems.id` |
+| `product_id` | integer | ID of the product. | [Get a product](/docs/rest-management/catalog/products#get-a-product) <br /> `data.id` |
 | `product_name` | string | Name of the product. | [Get a product](/docs/rest-management/catalog/products#get-a-product) <br /> `data.name` |
 | `purchase_price` | number | Final price of product that the shopper actually pays before taxes and other charges. Equals the `base_price` unless there is a `sale_price`. The price should include or exclude tax, based on the store settings. | [Get a product](/docs/rest-management/catalog/products#get-a-product) <br /> `data.price` or <br /> `data.sale_price` |
-| `quantity` | number | Number of items of product purchased. | [Get a cart](/docs/rest-management/carts/carts-single#get-a-cart) <br /> `data.line_items.physical_items.quantity` or <br /> `data.line_items.digital_items.quantity` |
+| `quantity` | integer | Number of items of product purchased. | [Get a cart](/docs/rest-storefront/carts#get-a-cart) <br /> `lineItems.physicalItems.quantity` or <br /> `lineItems.digitalItems.quantity` |
+| <sup>++</sup>`retail_price` | number | The retail price of the product. The price should include or exclude tax, based on the store settings. | [Get a product](/docs/rest-management/catalog/products#get-a-product) <br /> `data.retail_price` |
 | `sale_price` | number | Price of product after sale. The price should include or exclude tax, based on the store settings. | [Get a product](/docs/rest-management/catalog/products#get-a-product) <br /> `data.sale_price` |
 | `sku` | string | User-defined SKU for the product or variant (whichever is applied). | [Get a product](/docs/rest-management/catalog/products#get-a-product) <br /> `data.sku` or `data.variants.sku` |
-| `variant_id` | integer | ID of the variant. | [Get a product](/docs/rest-management/catalog/products#get-a-product) <br /> `data.variants.id` | 
+| <sup>+++</sup>`variant_id` | integer | ID of the variant. | [Get a product](/docs/rest-management/catalog/products#get-a-product) <br /> `data.variants.id` | 
```

Comment:
@bc-sanjayjoshi,
The same as with `product_id`. The internal doc says, it's a string, but it's actually an integer

Reply:
@bc-sanjayjoshi It seems like updating the internal doc would be the best fix, as Roman mentioned [here](https://github.com/bigcommerce/dev-docs/pull/1838/files#r1147814279). I'll go ahead and resolve the conversation so I can publish my doc